motionplan/planManager.go

```diff
@@ -54,6 +54,95 @@ func newPlanManager(
 	return &planManager{planner: p, frame: frame, useTPspace: len(frame.PTGSolvers()) > 0}, nil
 }
 
+// PlanMultiWaypoint plans a motion through multiple waypoints, using identical constraints for each
+// Unlike PlanSingleWaypoint, this does not break up individual.
+func (pm *planManager) PlanMultiWaypoint(ctx context.Context, request *PlanRequest, goals []spatialmath.Pose) (Plan, error) {
```

Comment:
Why not just change the PlanRequest destination to accept a slice?  Its confusing to me how the list of goals would interact with the request destination

Reply:
Since the features we are adding via DoCommand and `extra` are quick shortest-path fixes rather than fully fleshed-out updates, I'd prefer not to make major structural changes to public data structures until the feature is more formalized.

I may update this as part of RSDK-8835 but prefer not to do so now.

Reply:
Ok I'm good with this then, but just want to flag this as something that's suboptimal that we should go back and fix later on once we figure out more sustainable data structures. 

Maybe you could make a ticket for removing this function?

Reply:
Made RSDK-8850 as an umbrella for all of this.